easybuild/easyconfigs/l/Libint/Libint-1.1.4-intel-2016a.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Libint'
+version = '1.1.4'
+
+homepage = 'https://sourceforge.net/p/libint/'
+description = """Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
+ matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'opt': True, 'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ('http://sourceforge.net/projects/libint/files/v1-releases/', 'download')
+
+configopts = "--enable-deriv --enable-r12"
```

Comment:
please add `--enable-shared`, and also check for `['lib/lib%s.so' % x for x in ['deriv', 'int', 'r12']]` in sanity check